security-proxy/src/deb/resources/etc/georchestra/security-proxy/targets-mapping.properties

```diff
@@ -0,0 +1,11 @@
+analytics=http://localhost:8080/analytics/
+catalogapp=http://localhost:8080/catalogapp/
+downloadform=http://localhost:8080/downloadform/
+extractorapp=http://localhost:8080/extractorapp/
+geonetwork=http://localhost:8080/geonetwork/
+geoserver=http://localhost:8180/geoserver/
+geofence=http://localhost:8180/geofence/
+header=http://localhost:8080/header/
+ldapadmin=http://localhost:8080/ldapadmin/
+mapfishapp=http://localhost:8080/mapfishapp/
+static=http://localhost:8080/header/
```

Comment:
Yes it could actually ; the main reason I put that in here this way is that it is coherent with what our puppet recipe does internally at camptocamp. I first had a puppet/augeas code that was modifying these values, before removing it and replace the default values here accordingly.